docker-compose.yml

```diff
@@ -91,6 +91,8 @@ services:
         environment:
             POSTGRES_USER: ${PGUSER}
             POSTGRES_DB: ${PGDATABASE}
+            # https://github.com/docker-library/postgres/issues/681
```

Comment:
we use 10.x rather than 9.x, but it seems a general problem with recent versions of Postgres